### PR TITLE
Add nimbus_addPeer rpc call

### PR DESCRIPTION
**Pull request by hmel**
Adds a nimbus_addPeer rpc call which adds an enode to the list of trusted peers.

### CODE REVIEW

File: nimbus/rpc/common.nim [hunk 1]
```diff
@@ -57,3 +57,10 @@ proc setupCommonRpc*(node: EthereumNode, conf: NimbusConf, server: RpcServer) =
         listener: $enode.address.tcpPort
       )
     )
+
+  server.rpc("nimbus_addPeer") do(enode: string) -> bool:
+    var res = ENode.fromString(enode)
+    if res.isOk:
+      asyncCheck node.peerPool.connectToNode(newNode(res.get()))
```
Comment (kdeme) [inline nimbus/rpc/common.nim:64]: `asyncCheck` is deprecated, use `asyncSpawn` instead. Also be aware that when using these, the proc called must not raise, which I believe is fine here as connectToNode is mostly doing an rlpx connect.

File: nimbus/rpc/common.nim [hunk 2]
```diff
@@ -57,3 +57,10 @@ proc setupCommonRpc*(node: EthereumNode, conf: NimbusConf, server: RpcServer) =
...
+      return true
```
Comment (kdeme) [inline nimbus/rpc/common.nim:65]: Not sure what you would prefer as behaviour here, but `connectToNode` might fail. If you need to know this, you will have to `await` in this call to get that result.

Comment (hmel) [reply nimbus/rpc/common.nim:65]: This is just a request to add a node. We don't want to block the caller in an rpc call.

File: nimbus/rpc/common.nim (hunk 1 above)
Comment (hmel) [reply nimbus/rpc/common.nim]: Updated

**Review by kdeme — APPROVED** [review]

**Pull request merged**